test/modules/model/TinyLlamaWithFusedRMSNorm/model.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+from tico.serialize.operators.adapters.rmsnorm import patched_llama_rmsnorm
+from tico.utils.pytree_utils import register_dynamic_cache
+
+from transformers import AutoModelForCausalLM
+
+from test.modules.base import TestModuleBase
+
+
+class TinyLlamaWithFusedRMSNorm(TestModuleBase):
+    def __init__(self):
+        super().__init__()
+        with patched_llama_rmsnorm():
```

Comment:
How can we patch multiple modules? For example, how can we patch both LlamaRMSNorm and LlamaAttention?

Reply:
@seockho-kim Yes, I think the same way.